Makefile

```diff
@@ -541,7 +541,7 @@ nxOMSGenerateInventoryMof:
 
 nxOMSPlugin:
 	rm -rf output/staging; \
-	VERSION="3.52"; \
+	VERSION="3.53"; \
```

Comment:
@nisarawg - are you planning to relesase a new version of the nxOMSPlugin Resource as well.

Reply:
@Jinesh12 - The configuration files which are part of the nxOMSPlugin were also changed but no changes were made to the scripts. Does that not require a new version of nxOMSPlugin to be released?

Reply:
I think it would. Adding @hestolz - to review from nxOMSPlugin end.

Reply:
Config files are part of the module; you need to release a new version. The instructions are [here](https://microsoft.sharepoint.com/teams/WAG/EngSys/_layouts/OneNote.aspx?id=%2Fteams%2FWAG%2FEngSys%2FShared%20Documents%2FNotebooks%2FCloud%20Agent%20Team&wd=target%28OMS%20Linux.one%7C3F28D507-A5EB-471A-8411-45467E4DDF90%2FDSC%20Module%20Process%7CB602A3A6-F7C8-4D5C-82E3-C10DE746788B%2F%29).